ci/scripts/utils/ci_utils.sh

```diff
@@ -155,3 +154,24 @@ function publish_logs() {
     fi
     echo "${URL}"
 }
+
+function cleanup_experiment() {
+
+    local EXPDIR="$1"
+    local pslot
+    local ARCDIR
+    local ATARDIR
+    local DATAROOT
```

Comment:
Sorry, missed one spot:

```suggestion
    local DATAROOT
    local COMROOT
```